cli/cmd/install.go

```diff
@@ -96,6 +98,18 @@ type (
 		CrtExpiryAnnotation string
 	}
 
+	proxyInjectorValues struct {
+		*tlsValues
+	}
+
+	profileValidatorValues struct {
+		*tlsValues
```

Comment:
I decided to declare the `tlsValues` struct so that it can be used as the return type of the [`generateWebhookTLS()` function](https://github.com/linkerd/linkerd2/blob/49da3c8cc7383e9f8b94b2d11733156e4c4cd5fe/cli/cmd/install.go#L207-L217), instead of returning two strings. Yeah, I thought of omitting the nested type here, but then decided w/e it's easy enough to change, if it doesn't work out.